CONTRIBUTING.md

```diff
@@ -3,20 +3,56 @@ Feel free to contribute PRs, Issues, Documentation, or in any other way to this
 
 # Dev Environment Setup
 This application has a development environment that supports Docker. We believe this makes it easier to get started with the application.
+To get started, you will need either Docker [desktop](https://docs.docker.com/desktop/#download-and-install) or Docker [engine](https://docs.docker.com/engine/install) as well as
+Docker Compose which is a tool that is used for defining and running multi-container Docker applications. In our case for running the web application, database and selenium for testing,
 
 ## First time setup
-1. Setup web and db containers. `docker compose up -d`
-1. Run database migrations. `docker compose run --rm web bundle exec rake db:create`
-1. The application should be available at [localhost:3000](localhost:3000)
+1. Setup web and db containers. `docker-compose up -d`
+2. Create database tables. `docker-compose run --rm web bundle exec rake db:create`
+3. Run database migrations. `docker-compose run --rm web bundle exec rake db:migrate`
 
 ## Daily use
-Run `docker compose up -d` and the application should be available at [localhost:3000](localhost:3000).
+Run `docker-compose up -d` will run those services in the background and the application should be available at [localhost:3000](localhost:3000).
+Run `docker ps` to see all of the running services which are defined in `docker-compose.yml`.
 
 ## Testing
-Run `docker compose run --rm web bundle exec rspec` to run the test suite. We have GitHub Actions setup to run the test suite automatically. We expect new tests for new functionality, and your PR to having a passing test suite before review.
+Run `docker-compose run --rm web bundle exec rspec` to run the test suite. We have GitHub Actions setup to run the test suite automatically. We expect new tests for new functionality, and your PR to having a passing test suite before review.
 
 ## Teardown
-When you're done, run `docker compose down` to stop the containers. You can remove all containers, images, and volumes with `docker compose rm -v` when you're done contributing to ConfBuddies.
+When you're done, run `docker-compose down` to stop the containers. You can remove all containers, images, and volumes with `docker-compose rm -v` when you're done contributing to ConfBuddies.
+
+## Helpful Docker Commands
+To open the Rails console:\
+`docker-compose run --rm web bin/rails c`
+
+To open the Postgres console:\
+`docker-compose exec db psql -U postgres ConfBuddies_development`
+
+To run your tests:\
+`docker-compose run --rm web bundle exec rspec`
+
+To run rubocop:\
+`docker-compose run --rm web bundle exec rubocop`
+
+To reset the database with the seed data:
```

Comment:
We don't have any seed data at this time. It's a good idea to add some though.

Reply:
Would you like me to remove this for now? Or I could create an issue for seed data to be created? 😄

Reply:
Yeah!! Make an issue! That'd be super rad. I was about to do that anyway.

Reply:
Created an issue for this https://github.com/ChaelCodes/ConfBuddies/issues/66 👍